lib/stripe/subscriptions/coupon.ex

```diff
@@ -64,6 +66,7 @@ defmodule Stripe.Coupon do
                optional(:currency) => String.t(),
                optional(:max_redemptions) => pos_integer,
                optional(:metadata) => Stripe.Types.metadata(),
+               optional(:name) => String.t(),
```

Comment:
I think you can update a coupon with `name` as well

Reply:
Added `name` to update. Also changed the metadata there to optional, like in the other modules.